src/amber/environment/environment.cr

```diff
@@ -1,4 +1,4 @@
-module Amber
```

Comment:
`environment/environment.cr` Doesn't look good. Could we call it `environment/env.cr` or `environment/name.cr`

Reply:
I thought of this I will name it env